index.js

```diff
@@ -325,12 +325,11 @@ var MongodbDriver = Base.extend({
             else
               db.collection(collection).insertOne(options, {}, callbackFunction);
             break;
+          case 'removeMany':
```

Comment:
Should probably be called `deleteMany`. Makes more sense, no?

Reply:
Thanks for helping on the review :)

The other case is named remove, @marc0der probably oriented after that, so wouldn't be an issue. This is an internal method anyway, but I agree that the mongodb needs some love. If  you would be willing to give it a shot you're as always more than welcome to so :)

Reply:
Yes, exactly as @wzrdtales says.

@wzrdtales would you hit that merge button please :-) It's been open since 1 September 2017.

Reply:
@marc0der Nope. There are no tests, if you want you can create a fork from his tree and add them and put this into a new PR and I will he happy to merge :)

Reply:
Also the altering of remove needs to be removed as this would be a breaking change.

Reply:
BTW, to be clear this is not my PR; it was raised by @meengit. I merely require this functionality.

Reply:
I know that is why I told to fork :)

Reply:
Don't worry about it, I've already opted for using a different migration framework.

Reply:
That is totally ok, I always appreciate people contributing to open source though :)

Reply:
As do I, in fact I was using your open source project in my own [open source project](https://github.com/sdkman/sdkman-cli) ;-)

Reply:
Wasn't a critique though, I'm totally fine if anyone uses something else instead. Open Source means freedom, but shouldn't be considered to be "for free". Everyone and every comany should contribute back, unfortunately many do not. I ment actually this specific PR of @meengit here which I do highly appreciate and I just mentioned it b/c this has been open for a really long time, which I'm sorry for. And the same reason why this has been lying around is which makes me appreciate people that help. I do not have unlimited time, so everyone that contributes is a tremendous help. And that is quite true for every open source project that is not backed by money or a company :)